app/javascript/packages/document-capture/components/address-search.spec.tsx

```diff
@@ -25,20 +26,24 @@ describe('AddressSearch', () => {
 
   let server: SetupServerApi;
   before(() => {
+    global.window.fetch = fetch;
     server = setupServer(
+      rest.post(LOCATIONS_URL, (_req, res, ctx) => res(ctx.json([{ name: 'Baltimore' }]))),
       rest.post(ADDRESS_SEARCH_URL, (_req, res, ctx) => res(ctx.json(DEFAULT_RESPONSE))),
     );
     server.listen();
   });
 
   after(() => {
     server.close();
+    global.window.fetch = () => Promise.reject(new Error('Fetch must be stubbed'));
```

Comment:
I would not expect to see this, since #7518 changed the test initialization of `fetch`, and I think this line of code could have unexpected consequences for tests which occur after this one and rely on a real `fetch` implementation.

Was this added for any particular reason? Removing the `fetch`-related stubbing code here, the tests still pass for me.

Reply:
> This might have been a mistake. It looks like you cleaned this up [here](https://github.com/18F/identity-idp/pull/7564). Sorry about that...

All good! Thanks for confirming.